### PR TITLE
docs: update remaining rust guide

### PR DESCRIPTION
**Pull request by lwshang**
Update the pages of "multiply dependency", "profile".

The "multiply dependency" guide requires the latest master branch `dfx`. So we'd better publish this page after we have a new `dfx` release.

### CODE REVIEW

File: docs/modules/rust-guide/examples/mul-deps/deps-main.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ use ic_cdk::export::candid;
 #[import(canister = "multiply_deps")]
 struct CounterCanister;
 
-#[query]
+#[update]
```
Comment (krpeacock) [inline docs/modules/rust-guide/examples/mul-deps/deps-main.rs:7]: What's the reason for this change?

**Review by krpeacock — APPROVED** [review]
This looks good, although I haven't followed all the steps myself

File: docs/modules/rust-guide/examples/mul-deps/deps-main.rs (hunk 1 above)
Comment (lwshang) [reply docs/modules/rust-guide/examples/mul-deps/deps-main.rs:7]: Akhi clarified that "Inter canister query calls implementation in the replica is a poc and it'll be quite a while before we prioritize turning that into a complete feature".

File: docs/modules/rust-guide/pages/multiply-dependency.adoc [hunk 2]
```diff
@@ -21,7 +21,7 @@ For this tutorial, both canister smart contracts are in the same project.
 
 * The {native} canister creates an actor with a `+cell+` variable to contain the current value that results from an operation.
 * The `+mul+` function takes a natural number as input, multiplies the input value by three and stores the result in the `+cell+` variable.
-* The {lang} canister provides a `+read+` function that is a simple query that returns the current value of the `+cell+` variable.
+* The {lang} canister provides a simple `+read+` function that returns the current value of the `+cell+` variable. Notice that the function making inter-canister call should be a `+update+` method even though the method to be called is `+query+`.
```
[the remark this answers is not in the document]
Comment (lwshang) [reply docs/modules/rust-guide/pages/multiply-dependency.adoc:24]: This PR has been merged. Will include better explanation in another PR.

**Pull request merged**